README.md

```diff
@@ -55,13 +55,13 @@ SkyPilot **cuts your cloud costs**:
 
 SkyPilot supports your existing GPU, TPU, and CPU workloads, with no code changes.
 
-Install with pip:
+Install with pip (we recommend using the nightly build for the latest features/updates) or [from source](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html):
 ```bash
```

Comment:
Wdyta `aws,gcp,azure,oci,lambda,runpod,ibm,scp,kubernetes` everywhere?

Reply:
I changed it to `aws,gcp,azure,kubernetes,oci,lambda,runpod,ibm,scp`. Wdyt?

Reply:
Sounds good. Reordered. Thanks!